cyfronet-fid/marketplace
# [#142] Add Dotenv Gem

# What is in this PR?

This PR adds new dependency for test and development - dotenv (https://github.com/bkeepers/dotenv)

There has also been update to `README.md` mentioning it.

# How can I test it?

1. create `.env` file in the root application directory
2. add `MP_DATABASE_HOST="http://nodomain"`
3. run `bin/setup` it should fail due to wrong db connection
